### PR TITLE
Reposize-badge added

### PR DESCRIPTION
**Pull request by satyamkale27**
## Fixes Issue
#2116

## Changes proposed
repo size badge added that displays the rize of repo

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
reposize badge

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2223"></a>

### CODE REVIEW

**Comment by satyamkale27** [comment]
Please take a look @eddiejaoude

**Comment by eddiejaoude** [comment]
Thank you @satyamkale27 could you resolve the conflict please

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-[![Open in GitPod](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree) ![Uptime](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2FEddieHubCommunity%2Fmonitoring%2Fmaster%2Fapi%2Flink-free%2Fuptime.json) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![GitHub release (latest by date)](https://img.shields.io/github/v/release/EddieHubCommunity/LinkFree?style=for-the-badge)](https://github.com/EddieHubCommunity/LinkFree/releases)
+
```
Comment (schmelto) [inline README.md:1]:
```suggestion
```

**Review by schmelto — COMMENTED** [review]
further I would take a look at consistent badge styles

**Comment by satyamkale27** [comment]
> further I would take a look at consistent badge styles

**Comment by satyamkale27** [comment]
should i make changes  about consistent badge styles @schmelto

**Comment by schmelto** [comment]
> should i make changes  about consistent badge styles @schmelto 

Ig this could be done in another pr

**Comment by satyamkale27** [comment]
sure @schmelto

**Review by schmelto — APPROVED** [review]

**Pull request closed without merging**